Packs/HealthCheck/Scripts/HealthCheckAnalyzeLargeInvestigations/HealthCheckAnalyzeLargeInvestigations.py

```diff
@@ -35,32 +35,38 @@ def format_dict_keys(data: List[Dict[str, Any]]) -> List:
 def main(args):
     thresholds = args.get('Thresholds', THRESHOLDS)
     prev_month = datetime.today() + dateutil.relativedelta.relativedelta(months=-1)
-
+    current_month = datetime.today()
     res = execute_command('GetLargestInvestigations', {
         'from': prev_month.strftime('%Y-%m-%d'),
-        'to': prev_month.strftime('%Y-%m-%d'),
+        'to': current_month.strftime('%Y-%m-%d'),
         'table_result': 'true',
     })
 
-    res_data = format_dict_keys(res['data'])
-    incidentsbiggerthan1mb = res_data
-    numberofincidentsbiggerthan1mb = res['total']
-
-    incidentsbiggerthan10mb = [incident for incident in res_data if int(incident['size'].split()[0]) > 10]
+    incidentsbiggerthan1mb = []
+    incidentsbiggerthan10mb = []
+    incidentswithmorethan500entries = []
+    for incident in res['data']:
+        if incident['AmountOfEntries'] >= 500:
+            incidentswithmorethan500entries.append(incident)
+        if round(incident['Size(MB)']) >= 10:
+            incidentsbiggerthan10mb.append(incident)
+        else:
+            incidentsbiggerthan1mb.append(incident)
```

Comment:
this means that only incidents larger than 1MB **and** smaller than 10MB will be considered as over 1MB.
for example, incident with size 15MB will only be counted once (in the over 10MB list).

is that ok?

Reply:
correct.